orchagent/flexcounterorch.cpp

```diff
@@ -393,39 +418,59 @@ bool FlexCounterOrch::getWredPortCountersState() const
     return m_wred_port_counter_enabled;
 }
 
-bool FlexCounterOrch::bake()
+bool FlexCounterOrch::isCreateOnlyConfigDbBuffers() const
 {
-    /*
-     * bake is called during warmreboot reconciling procedure.
-     * By default, it should fetch items from the tables the sub agents listen to,
-     * and then push them into m_toSync of each sub agent.
-     * The motivation is to make sub agents handle the saved entries first and then handle the upcoming entries.
-     * The FCs are not data plane configuration required during reconciling process, hence don't do anything in bake.
-     */
-
-    return true;
+    return m_createOnlyConfigDbBuffers;
 }
 
-static bool isCreateOnlyConfigDbBuffers(Table& deviceMetadataConfigTable)
+void FlexCounterOrch::handleDeviceMetadataTable(Consumer &consumer)
 {
-    std::string createOnlyConfigDbBuffersValue;
+    SWSS_LOG_ENTER();
 
-    try
+    auto it = consumer.m_toSync.begin();
+    while (it != consumer.m_toSync.end())
     {
-        if (deviceMetadataConfigTable.hget("localhost", "create_only_config_db_buffers", createOnlyConfigDbBuffersValue))
+        KeyOpFieldsValuesTuple t = it->second;
+        string key = kfvKey(t);
+        string op = kfvOp(t);
+        auto data = kfvFieldsValues(t);
+
+        // Only process localhost entries
+        if (key == "localhost" && op == SET_COMMAND)
         {
-            if (createOnlyConfigDbBuffersValue == "true")
+            for (auto valuePair : data)
             {
-                return true;
+                const auto &field = fvField(valuePair);
+                const auto &value = fvValue(valuePair);
+
+                if (field == "create_only_config_db_buffers")
```

Comment:
@stephenxs Do you have yang model changes for this new field("create_only_config_db_buffers") in DEVICE_METADATA?

Reply:
If you already have the PR, please add that in description for reference.

Reply:
@kperumalbfn now new. it's there for years :)
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-device_metadata.yang#L251